docs/concepts/credentials/1_ctypes.md

```diff
@@ -39,6 +39,21 @@ When making a claim for a CType, all the following properties are required:
 }
 ```
 
+The CType schema is afterwards wrapped into the full CType object:
+
+```js title="CType example"
+{
+  schema:  {...} // as defined in the example above>,
+  owner: null, // DID of the owner, or null
+  hash: '0xda3861a45e0197f3ca145c2c209f9126e5053fas503e459af4255cf8011d51010' // For looking up on-chain
+}
+```
+
+## Nested CTypes
+
+A Nested CType is a hierarchical composite schema that includes other simpler CTypes as substructures. The Nested CType references the included sub-CTypes. For example, a company could use a Nested CType that includes the required credentials, qualifications, health and safety certs, etc of its current employees.
```

Comment:
Do we want to capitalise also Nested?

```suggestion
A Nested CType is a hierarchical composite schema that includes and references other simpler CTypes as substructures. For example, a company could use a Nested CType that includes the required credentials, qualifications, health and safety certs, etc of its current employees.
```

Reply:
I am not sure about capitalisation. I felt like every KILT Concept we have could be capitalised, because they act as proper nouns. But maybe we just align that in the end.